src/care/host_device_ptr.h

```diff
@@ -438,6 +438,7 @@ namespace care {
    void using_host_device_ptr_outside_of_raja_loop_not_allowed(T foo); 
 } // namespace care
 
+#if defined(CARE_ENABLE_MANAGED_PTR)
```

Comment:
Sorry, I may be missing something, but why are you putting this here? We are declaring getRawPointers with

care::host_device_ptr<T>

not managed_ptr , so it should be a different case?

Reply:
The original getRawPointers functions are defined in managed_ptr.hpp. So if managed_ptr support is disabled in CARE, there is no reason for this function.